nodejs/week3/classExercise/src/backend/api/concerts.js

```diff
@@ -0,0 +1,145 @@
+// const { request, response } = require("express");
+const express = require("express");
```

Comment:
The homework only asked for the calculator and the backend for the meal sharing app, so please remove all the concert related files.